plugin/evm/version_test.go

```diff
@@ -17,13 +18,19 @@ const compatibilityFile = "../../compatibility.json"
 
 func TestCompatibility(t *testing.T) {
 	compat, err := os.ReadFile(compatibilityFile)
-	assert.NoError(t, err)
+	require.NoError(t, err, "reading compatibility file")
 
 	var parsedCompat rpcChainCompatibility
 	err = json.Unmarshal(compat, &parsedCompat)
-	assert.NoError(t, err)
+	require.NoError(t, err, "json decoding compatibility file")
 
 	rpcChainVMVersion, valueInJSON := parsedCompat.RPCChainVMProtocolVersion[Version]
-	assert.True(t, valueInJSON)
-	assert.Equal(t, rpcChainVMVersion, version.RPCChainVMProtocol)
+	if !valueInJSON {
+		t.Fatalf("%s has subnet-evm version %s missing from rpcChainVMProtocolVersion object",
+			filepath.Base(compatibilityFile), Version)
+	}
+	if rpcChainVMVersion != version.RPCChainVMProtocol {
+		t.Fatalf("%s has subnet-evm version %s stated as compatible with RPC chain VM protocol version %d but AvalancheGo protocol version is %d",
+			filepath.Base(compatibilityFile), Version, rpcChainVMVersion, version.RPCChainVMProtocol)
+	}
```

Comment:
nit: why did we change these from require/assert to fatal? require methods ending with `f` (like `Equalf`) support printf messages with args.

Reply:
It's to have a single line error message instead of a bunch of lines produced by `require.Equalf`, I think.